### PR TITLE
Split anal_full_process_single_seed.sh into smaller bash scripts.

### PR DESCRIPTION
**Pull request by Jacob608**

### CODE REVIEW

File: Simulation_Analysis/individual_frame_network_analysis.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#SBATCH --job-name="Ana R1-0.01erate"
+#SBATCH -A p31412
+#SBATCH -p short    ## partition
```
Comment (XinyanYang95) [inline Simulation_Analysis/individual_frame_network_analysis.sh]: we may need a normal job

File: Simulation_Analysis/individual_frame_network_analysis.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#SBATCH -N 1  ## number of nodes
+#SBATCH -n 1  ## number of cores
+#SBATCH --output=R-%x.%j.out
+#SBATCH --mem=30G
+#SBATCH --ntasks-per-node=1  ## number of cores
+#SBATCH -t 00:05:00
```
Comment (XinyanYang95) [inline Simulation_Analysis/individual_frame_network_analysis.sh]: 47:50:00

**Review by XinyanYang95 — CHANGES_REQUESTED** [review]

File: Simulation_Analysis/individual_frame_network_analysis.sh (hunk 1 above)
Comment (Jacob608) [reply Simulation_Analysis/individual_frame_network_analysis.sh]: done

File: Simulation_Analysis/individual_frame_network_analysis.sh (hunk 2 above)
Comment (Jacob608) [reply Simulation_Analysis/individual_frame_network_analysis.sh]: done

**Pull request merged**